apis/management.cattle.io/v3/authz_types.go

```diff
@@ -52,6 +54,7 @@ type ProjectSpec struct {
 	ClusterName                   string                  `json:"clusterName,omitempty" norman:"required,type=reference[cluster]"`
 	ResourceQuota                 *ProjectResourceQuota   `json:"resourceQuota,omitempty"`
 	NamespaceDefaultResourceQuota *NamespaceResourceQuota `json:"namespaceDefaultResourceQuota,omitempty"`
+	EnableProjectMonitoring       *bool                   `json:"enableProjectMonitoring" norman:"default=false"`
```

Comment:
why is this a pointer?

Reply:
similar to `Cluster` struct.

Reply:
We only do pointer bools when we _need_ to. That generally is to support backwards compatibility and upgrade scenarios. If it doesn't have a functional need to be a pointer, it shouldn't be.

Reply:
@cjellick , Got.